x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/utils/indexing_service.ts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
There is only a single file under utils. How about removing the folder and just putting in new_vector_layer_wizard folder. Can you also rename to a more descriptive name like create_new_index_pattern?